clients/vscode/package.json

```diff
@@ -148,7 +148,7 @@
             "default": "",
             "pattern": "(^$)|(^https?:\\/\\/\\S+$)",
             "patternErrorMessage": "Please enter a validate http or https URL.",
-            "markdownDescription": "Specify API Endpoint of Tabby.  \nIf leave empty, server endpoint in [Tabby Agent Settings](command:tabby.openTabbyAgentSettings) will be used."
+            "markdownDescription": "**Specify API Endpoint of Tabby**  \nIf leave empty, server endpoint in [Tabby Agent Settings](command:tabby.openTabbyAgentSettings) will be used.  \n\n**Set Up Authentication Credentials**  \nTo access the API endpoint, please [Set Credentials](command:tabby.setApiToken)."
```

Comment:
Let's update the description like this, as the authentication is optional.

Specify the API endpoint for the Tabby server.
If authentication is required, please [Set Credentials].
If left empty, the server endpoint specified in the [Tabby Agent Settings] will be used.

```suggestion
            "markdownDescription": "Specify the API endpoint for the Tabby server.\nIf authentication is required, please [Set Credentials](command:tabby.setApiToken).\nIf left empty, the server endpoint specified in the [Tabby Agent Settings](command:tabby.openTabbyAgentSettings) will be used."
```

Reply:
Updated, it looks like only `  \n` can change the line.
![1](https://github.com/TabbyML/tabby/assets/5305874/8148f206-2d06-4273-bea2-4cffc44e5320)
Screen record: https://jam.dev/c/2a78f085-4b44-4123-a35b-204dcfa95760